specification/trace/semantic_conventions/messaging.md

```diff
@@ -217,6 +217,23 @@ For Apache Kafka producers, [`peer.service`](./span-general.md#general-remote-se
 The `service.name` of a Consumer's Resource SHOULD match the `peer.service` of the Producer, when the message is directly passed to another service.
 If an intermediary broker is present, `service.name` and `peer.service` will not be the same.
 
+#### CloudEvents
+
+For CloudEvents clients, the following attributes are defined:
+
```

Comment:
Is there any guidance you could give here on how to fill the generic attributes defined further above?
Is there any overlap with the custom attributes you're adding here?

Reply:
> Is there any guidance you could give here on how to fill the generic attributes defined further above?

You mean the generic ones that are marked as required? I'm not sure how all the others even relate to cloudevents, but things like `messaging.system` and `messaging.destination` should be possible to find out. I'll see what I can add. 


> Is there any overlap with the custom attributes you're adding here

- `event_id`: This could be `message_id`, but I'm not sure if changing the nomenclature is good, since cloud events uses `event` for everything as far as I could see.

Reply:
I think I'll leave for now `event_id` and when the upstream PR is there, I'll post in the cloudevents spec slack channel, asking for someone to take a look if what we have makes sense.